### PR TITLE
Header: Style header with a white background color except for the home page where the background should be transparent.

### PR DESCRIPTION
**Pull request by coolsoftwaretyler**
The header background is white on all pages, except the home page, in which it renders transparent.

### CODE REVIEW

File: client/app/layout/header/header.component.js [hunk 1]
```diff
@@ -1,9 +1,18 @@
+function backgroundSelector () {
+	if (window.location.pathname == "/"){
```
Comment (enkhi77) [inline client/app/layout/header/header.component.js]: Go ahead and inject $state and use state.name.  Also use the triple eq for this please.

File: client/app/layout/header/header.component.js [hunk 2]
```diff
@@ -1,9 +1,18 @@
+function backgroundSelector () {
```
Comment (enkhi77) [inline client/app/layout/header/header.component.js]: For the class, we can set this via the angular lifecycle.  Please see me about this one and we can talk this out.

**Review by enkhi77 — COMMENTED** [review]
Some comments in the code, please talk to me about the changes.

**Comment by coolsoftwaretyler** [comment]
Thanks, Wes! I had to run to a family event. Should I fix later tonight?
It's cool if you want to assign to someone else who is still around.

**Comment by enkhi77** [comment]
Go ahead and work on it tonight.  I should be online tonight and will be able to work with you on how to code out the controller

**Comment by coolsoftwaretyler** [comment]
Hey Wes, I'm having some trouble injecting $state. Let me know if you get a chance tonight to help walk me through it. I'll keep plugging away at it.

**Comment by coolsoftwaretyler** [comment]
Because of the animation on the home page we will want the text to have some sort of glow or better readability.

**Comment by coolsoftwaretyler** [comment]
Marking this as done because I think @enkhi77 sent in a PR two weeks ago.

**Pull request closed without merging**